$OpenBSD: patch-zurmo_app_protected_modules_install_utils_CheckServicesUtil_php,v 1.1 2013/02/05 16:54:32 ajacoutot Exp $

Don't check for the webserver, both in-tree httpd(8) and nginx(8) work fine.

--- zurmo/app/protected/modules/install/utils/CheckServicesUtil.php.orig	Tue Feb  5 17:52:21 2013
+++ zurmo/app/protected/modules/install/utils/CheckServicesUtil.php	Tue Feb  5 17:52:34 2013
@@ -42,8 +42,7 @@
 
         private static function getServicesToCheck()
         {
-            return array('WebServer',
-                         'Php',
+            return array('Php',
                          'PhpTimeZone',
                          'PhpMemoryBytes',
                          'PhpFileUploads',
